### PR TITLE
Add new neuron 7.9.0a and remove older neurodamus packages

### PR DESCRIPTION
**Pull request by pramodk**

### CODE REVIEW

**Review by ferdonline — DISMISSED** [review]
Didn’t know there is an official 7.8.1 tag.
LGTM

**Review by matz-e — DISMISSED** [review]

**Comment by pramodk** [comment]
retest this please

File: var/spack/repos/builtin/packages/neuron/package.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Neuron(CMakePackage):
     patch("fix_brew_py_18e97a2d.patch", when="@7.8.0c")
 
     version("develop", branch="master")
-    version("7.9.0a",  commit="acdd898", preferred=True)
+    version("7.9.0a",  commit="fc74b85", preferred=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/neuron/package.py:35]: Did you delete the PR directory?

Comment (pramodk) [reply var/spack/repos/builtin/packages/neuron/package.py:35]: you see - I am forgetting things :expressionless:! thanks!

**Comment by pramodk** [comment]
retest this please

**Review by ferdonline — APPROVED** [review]

**Pull request merged**